src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperDescriptorResolver.cs

```diff
@@ -89,7 +97,7 @@ public IEnumerable<TagHelperDescriptor> Resolve([NotNull] TagHelperDescriptorRes
                         directiveDescriptor.Location,
                         Resources.FormatTagHelperDescriptorResolver_EncounteredUnexpectedError(
                             "@" + directiveName,
-                            directiveDescriptor.LookupText,
+                            directiveDescriptor.DirectiveText,
```

Comment:
ask I ask later -- why reuse the same `TagHelperDirectiveDescriptor` type?  you're mixing concepts and then the code that consumes it has to split things again.